ResourceStatusSystem/Utilities/CSHelpers.py

```diff
@@ -256,7 +256,7 @@ def getFTS():
 
   #FIXME: FTS2 will be deprecated (first 2 lines that follow)
   ftsEndpoints = gConfig.getValue('Resources/FTSEndpoints/Default/FTSEndpoint', [])
-  ftsEndpoints =+ _getFTSEndpoints('Resources/FTSEndpoints/FTS2')
+  ftsEndpoints += _getFTSEndpoints('Resources/FTSEndpoints/FTS2')
```

Comment:
This is fixed also in another PR, I am surprised it is still not in integration branch.